src/main/java/frc/robot/commands/MoveArmCommand.java

```diff
@@ -0,0 +1,58 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import javax.swing.plaf.basic.BasicInternalFrameTitlePane.MoveAction;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.HardwareMap;
+import frc.robot.subsystems.Intake;
+
+
+public class MoveArmCommand extends CommandBase {
+  private XboxController m_controller;
+  private Intake m_intakeSubsystem;
+  /**
+   * Creates a new MoveArmCommand.
+   */
+  public MoveArmCommand(XboxController controller, Intake intake) {
```

Comment:
take in a DoubleSupplier instead of an xbox controller

Reply:
Why? Explain this to me + Tia + Ethan today during the meeting